fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -814,4 +814,41 @@ public int searchCount(Select countQuery) throws FHIRPersistenceDataAccessExcept
         }
         return resourceIds;
    }
+
+    @Override
+    public Map<Integer, List<Long>> searchWholeSystem(Select wholeSystemQuery) throws FHIRPersistenceDataAccessException,
+            FHIRPersistenceDBConnectException {
+        final String METHODNAME = "searchWholeSystem";
+        log.entering(CLASSNAME, METHODNAME);
```

Comment:
worth adding the argument to this?

Reply:
The SQL actually gets logged by QueryUtil.prepareSelect(), so I chose not to add here